app/services/change_schedule.rb

```diff
@@ -9,6 +9,7 @@ class ChangeSchedule
   attribute :course_identifier
   attribute :schedule_identifier
   attribute :cohort, :integer
+  attribute :attempt_to_change_cohort_leaving_billable_declarations, :boolean, default: false
```

Comment:
qq around participants moving from 2024 to 2021 outside the `attempt_to_change_cohort_leaving_billable_declarations`, do we intend to keep that available, I remember we discussed it but not sure what we landed on?

Reply:
I'm not sure on this yet; Nathan caught up with Mili on this at the end of last week and the upshot was that we may not let providers move participants from 2021 -> 2024 using this mechanism initially and instead it would be something we will explore after registration opens. I think this flag may change to only prevent 2021 -> 2024 and we end up allowing 2024 -> 2021 to be triggered by providers, but its still TBC.